x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/components/additional_options_fields.tsx

```diff
@@ -183,26 +183,30 @@ export const AdditionalOptionsFields: React.FC<AdditionalOptionsFieldsProps> = (
       <EuiPanel hasBorder={true}>
         {optionalProviderFormFields.length > 0 ? (
           <>
-            <EuiTitle size="xxs" data-test-subj="provider-optional-settings-label">
-              <h4>
-                <FormattedMessage
-                  id="xpack.inferenceEndpointUICommon.components.additionalInfo.providerOptionalSettingsLabel"
-                  defaultMessage="Service settings"
-                />
-              </h4>
-            </EuiTitle>
-            <EuiText
-              css={css`
-                font-size: ${xsFontSize};
-                color: ${euiTheme.colors.textSubdued};
-              `}
-            >
-              <FormattedMessage
-                id="xpack.inferenceEndpointUICommon.components.additionalInfo.providerOptionalSettingsHelpLabel"
-                defaultMessage="Configure the inference provider. These settings are optional provider settings."
-              />
-            </EuiText>
-            <EuiSpacer size="m" />
+            {config.provider === ServiceProviderKeys.elasticsearch ? null : (
+              <>
+                <EuiTitle size="xxs" data-test-subj="provider-optional-settings-label">
+                  <h4>
+                    <FormattedMessage
+                      id="xpack.inferenceEndpointUICommon.components.additionalInfo.providerOptionalSettingsLabel"
+                      defaultMessage="Service settings"
+                    />
+                  </h4>
+                </EuiTitle>
+                <EuiText
+                  css={css`
+                    font-size: ${xsFontSize};
+                    color: ${euiTheme.colors.textSubdued};
+                  `}
```

Comment:
nit:
```suggestion
                  css={{
                    font-size: ${xsFontSize};
                    color: ${euiTheme.colors.textSubdued};
                  }}
```

Reply:
Updated in ebff5bc61d7b5212ac76edd658fff6c6f2a44c42